C/cp/Activity_selection.c

```diff
@@ -0,0 +1,81 @@
+// C Code for Activity Selection
+//Greedy Algorithm approach
+#include <stdio.h>
+#include <conio.h>
+//swap function to check and swap values
+void swap(int *x,int *y)
+{
+	int temp;
+	temp=*x;
+	*x=*y;
+	*y=temp;
+}
+//Main function start
+int main()
+{
+	int num,start[20],finish[20],i,j,prev=0,count=1;
```

Comment:
Use Dynamic memory allocation for arrays

Reply:
Done @siddharth25pandey kindly review it.